designcompose/src/main/java/com/android/designcompose/Layout.kt

```diff
@@ -283,7 +287,10 @@ internal fun calcLayoutInfo(
                     },
                 selfModifier = modifier,
                 marginModifier = marginModifier,
-                padding = style.layout_style.padding,
+                padding =
+                    style.layout_style.padding.orElseThrow {
```

Comment:
Can these be functions so that the `orElseOrThrow` stays in the function instead of here